### PR TITLE
add e2e

### PR DESCRIPTION
**Pull request by xuezhaojun**
Signed-off-by: xuezhaojun <zxue@redhat.com>

### CODE REVIEW

File: test/e2e/cluster_proxy_test.go [hunk 1]
```diff
@@ -1,9 +1,102 @@
...
+			err = exec.Stream(remotecommand.StreamOptions{
+				Stdin:  nil,
+				Stdout: &stdout,
+				Stderr: &stderr,
+				Tty:    false,
+			})
+			Expect(err).To(BeNil())
+			Expect(strings.Contains(stdout.String(), "hello")).To(Equal(true))
+		})
+	})
```
Comment (skeeey) [inline test/e2e/cluster_proxy_test.go:101]: We may need an unauthorized case

Comment (xuezhaojun) [reply test/e2e/cluster_proxy_test.go:107]: Sure, added one.

File: test/e2e/cluster_proxy_test.go [hunk 2]
```diff
@@ -1,9 +1,108 @@
...
+				Expect(err).ToNot(BeNil())
+				Expect(strings.Contains(err.Error(), "forbidden")).To(Equal(true))
+			})
+		})
+
+		Context("URL is valid, but using unauth token", func() {
+			It("should return forbidden", func() {
+				_, err := clusterProxyUnAuthClient.CoreV1().Pods(hubInstallNamespace).List(context.Background(), v1.ListOptions{})
+				Expect(err).ToNot(BeNil())
+				Expect(strings.Contains(err.Error(), "Unauthorized")).To(Equal(true))
```
Comment (skeeey) [inline test/e2e/cluster_proxy_test.go:44]: may using errors.IsUnauthorized

Comment (xuezhaojun) [reply test/e2e/cluster_proxy_test.go]: Done

**Pull request merged**